rosdep/base.yaml

```diff
@@ -932,6 +933,7 @@ gstreamer0.10-plugins-ugly:
     wily: [gstreamer0.10-plugins-ugly]
 gstreamer0.10-pocketsphinx:
   fedora: [pocketsphinx-plugin]
+  gentoo: [app-accessibility/pocketsphinx]
```

Comment:
After curious wandering around from esound bug, I shall note that gstreamer0.10 using pocketsphinx will be removed from Gentoo as soon as I finally get around to solving it. gstreamer0.10 is known security vulnerable and it is irresponsible to pull it in.